aurora/aurora-configuration/src/main/config/data/templateRepository/auroraspacehomepage/data.xml

```diff
@@ -23,6 +27,14 @@
             <name>values</name>
             <value lang="fr">Oui##Non</value>
         </parameter>
+        <parameter>
+            <name>values</name>
+            <value lang="en">Yes##No</value>
+        </parameter>
+        <parameter>
+            <name>cols</name>
```

Comment:
What is this new parameter `cols`? And where is the `values` parameter in German?

Reply:
Missing again the German translation of `value` here